api/v1alpha1/enterprisecontractpolicy_types.go

```diff
@@ -90,6 +90,24 @@ type EnterpriseContractPolicyExceptions struct {
 	NonBlocking []string `json:"nonBlocking,omitempty"`
 }
 
+// EnterpriseContractPolicyConfiguration configuration of modifications to policy evaluation
+type EnterpriseContractPolicyConfiguration struct {
+	// ExcludeRules set of policy exclusions that, in case of failure, do not block
+	// the success of the outcome.
+	// +optional
+	// +listType:=set
+	ExcludeRules []string `json:"excludeRules,omitempty"`
+	// IncludeRules set of policy inclusions that are added to the policy evaluation.
+	// These override excluded rules.
+	// +optional
+	// +listType:=set
+	IncludeRules []string `json:"includeRules,omitempty"`
+	// Collections set of predefined rules.
+	// +optional
+	// +listType:=set
+	Collections []string `json:"collection;omitempty"`
```

Comment:
A bit late to review sorry, but FYI anyhow, I'm pretty sure the rego expects just one value here, rather than a list, so there's a mismatch between this and what needs to go into the config data.json file. (A list might be better come to think of it, so maybe we could tweak the rego to expect a list.)

Reply:
Also, if it is a list then the json key might be better pluralzied, i.e. "collections" instead of "collection".

Reply:
My expectation was that we'd be able to pass multiple collections that should be evaluated. I'll leave this as a collection and file a story to update the rego.